### ISSUE TITLE
Automatically open atwho-view

### ISSUE THREAD

**Issue by ebrucki** [ichord/At.js#387]
Is it possible to display the atwho-view / list without the user typing or clicking into the field, just like the At.js demo does? When you open the demo, the list is already there.
In my case, an inputor is inserted with js and the list should also open at that moment … it doesn't

**Comment by ichord** [comment]
there is a [.run](https://github.com/ichord/At.js/wiki/APIs#run)  API

**Comment by ebrucki** [comment]
Yes, I found out. It just doesn't work for contenteditable and since it isn't well documented what .run does I wasn't sure. It would be awesome if it would work for contenteditable.
I don't exactly remember my problem but something in the editableController - I think catchQuery - needed an event which it didn't get an threw an error.
It's the same problem when modifying the demo to .run in the contenteditable div.

**Comment by yibuyisheng** [comment]
I change the `jquery.atwho.js`:

![image](https://cloud.githubusercontent.com/assets/2581682/26163677/e9260af6-3b5c-11e7-9108-7665e8c24c75.png)

It worked temporarily!

But still wait the standard way to solve this bug.

**Comment by yasir-rafiq** [comment]
any update on it?

**Comment by omarmeky** [comment]
bump
